build-rr.sh

```diff
@@ -350,7 +360,8 @@ doextras || die 'platforms extras failed.  tillerman needs tea?'
 ln -sf ${PLATFORMDIR}/rump ./rumprun
 
 # do final build of the platform bits
-( cd ${PLATFORMDIR} && ${MAKE} || exit 1)
+${KERNONLY} && MAKEARGS="kernonly"
+( cd ${PLATFORMDIR} && ${MAKE} ${MAKEARGS} || exit 1)
```

Comment:
Can this bit somehow go into config.mk so that you don't need to keep typing "make kernonly" all the time in platform/hw?